games-emulation/ryujinx/files/ryujinx-0003-use-configurationpath-for-lo.patch

```diff
@@ -0,0 +1,86 @@
+From 78e443211b8dab8c0cf62ba5aada309ad47b9de9 Mon Sep 17 00:00:00 2001
+From: Andrew Udvare <audvare@gmail.com>
+Date: Tue, 22 Nov 2022 07:52:55 -0500
+Subject: [PATCH 3/3] Use ConfigurationPath for log base path
+
+---
+ src/Ryujinx.Ava/Program.cs                          | 2 +-
+ src/Ryujinx.Ui.Common/Configuration/LoggerModule.cs | 7 +++++--
+ src/Ryujinx/Program.cs                              | 2 +-
+ src/Ryujinx/Ui/MainWindow.cs                        | 2 +-
+ 4 files changed, 8 insertions(+), 5 deletions(-)
+
+diff --git a/src/Ryujinx.Ava/Program.cs b/src/Ryujinx.Ava/Program.cs
+index cc062a25..e75b0324 100644
+--- a/src/Ryujinx.Ava/Program.cs
++++ b/src/Ryujinx.Ava/Program.cs
+@@ -90,7 +90,7 @@ namespace Ryujinx.Ava
+             ConfigurationState.Initialize();
+ 
+             // Initialize the logger system.
+-            LoggerModule.Initialize();
++            LoggerModule.Initialize(AppDataManager.BaseDirPath);
```

Comment:
`AppDataManager.BaseDirPath` = `~/.config/Ryujinx`. `Logs` is appended later. I would prefer this be `~/.local/state/Ryujinx` since it's logs but that's more work to get going.